### PR TITLE
Moving from one IG w multiple AZs to one IG per AZ

### PR DESCRIPTION
**Pull request by Globegitter**
As discussed in #760

### CODE REVIEW

**Comment by robinpercy** [comment]
Awesome, thanks @Globegitter :)
/assign
/ok-to-test
/lgtm

File: docs/instance_groups.md [hunk 1]
```diff
@@ -119,6 +119,23 @@ So the procedure is:
 * Apply: `kops update cluster <clustername> --yes`
 * (no instances need to be relaunched, so no rolling-update is needed)
 
+
+## Moving from one instancs group spanning multiple AZs to one instance group per AZ
```
Comment (chrislovecnm) [inline docs/instance_groups.md]: Nit pick

> instancs

Probably should be "instance".

**Comment by justinsb** [comment]
LGTM after typo fix - thanks @Globegitter :-)

**Pull request merged**